primenet.py

```diff
@@ -3474,6 +3474,19 @@ def parse_work_unit_mfakto(filename, p):
 
     return counter, avg_msec_per_iter, pct_complete
 
+STAGES_RE = re.compile(r"^\s*Stages\s*=\s*([01])\s*(?:$|#)")
+
+
+def get_stages_mfaktx_ini(adir, filename):
+    ini_file = os.path.join(adir, filename)
+    # assume Stages=1 if not able to be found, because stages = 1 gives assignments that work for Stages=0
+    stages = 1
+    # configparser doesn't nicely work with .ini files without sections
+    for line in readonly_list_file(ini_file):
+        match = STAGES_RE.fullmatch(line)
```

Comment:
Unfortunately, `fullmatch()` [requires Python 3.4+](https://docs.python.org/3/library/re.html#re.Pattern.fullmatch). However, it should not be needed here due to the `^` and `$` already in your regular expression and the current code would fail if there was a comment on the same line due to not matching the full comment, so you would probably want to use `match()` anyway.